pkg/cmd/roachtest/roachtestutil/mixedversion/steps.go

```diff
@@ -249,43 +261,59 @@ func (s setClusterSettingStep) Run(
 		args = []interface{}{val}
 	}
 
-	return h.ExecWithGateway(rng, nodesRunningAtLeast(s.minVersion, h), stmt, args...)
+	return serviceByName(h, s.virtualClusterName).ExecWithGateway(
+		rng, nodesRunningAtLeast(s.virtualClusterName, s.minVersion, h), stmt, args...,
+	)
 }
 
 // resetClusterSetting resets cluster setting `name`.
 type resetClusterSettingStep struct {
-	minVersion *clusterupgrade.Version
-	name       string
+	minVersion         *clusterupgrade.Version
+	name               string
+	virtualClusterName string
 }
 
 func (s resetClusterSettingStep) Background() shouldStop { return nil }
 
 func (s resetClusterSettingStep) Description() string {
-	return fmt.Sprintf("reset cluster setting %q", s.name)
+	return fmt.Sprintf("reset cluster setting %q on %s tenant", s.name, s.virtualClusterName)
 }
 
 func (s resetClusterSettingStep) Run(
 	ctx context.Context, l *logger.Logger, rng *rand.Rand, h *Helper,
```

Comment:
Nit: neither `ctx` nor `l` is currently used by `Run`.

Reply:
Yes, these steps implement an interface, so they need to take those parameters even if they don't use it.